### DISCUSSION TITLE
Why is no native package (like `.RPM`, for Fedora) offerred?

### DISCUSSION THREAD

**Discussion by beedell-roke** [alainm23/planify#1492] [Q&A] [answered]
I am surprised that you solely offer it via Flatpak.

**Comment by alainm23** [comment, marked as answer]
Flatpak is the official distribution channel for Planify, and it works great on Fedora — it's actually pre-installed on most Fedora editions!

`flatpak install flathub io.github.alainm23.planify`

Maintaining native RPM packages would require significant ongoing effort to keep up with Fedora's release cycle, handle dependencies, and go through the Fedora review process. Flatpak already solves this with better sandboxing and automatic updates via Flathub.

Closing this, but feel free to reach out if you have issues with the Flatpak installation!

**Reply by beedell-roke** [reply]
@alainm23, thanks. I'll close this, considering that you've answered:

> Maintaining native RPM packages would require significant ongoing effort to keep up with Fedora's release cycle, handle dependencies, and go through the Fedora review process.

However, regarding the undermentioned:

> Flatpak is the official distribution channel for Planify, and it works great on Fedora — it's actually pre-installed on most Fedora editions!
> 
> `flatpak install flathub io.github.alainm23.planify`
> 
> Flatpak already solves this with better sandboxing and automatic updates via Flathub.

I'm familiar with how to install it, as I would hope that my work at [`issues/196`](https://github.com/alainm23/planify/issues/1965#issue-3551209914) demonstrates.

To elaborate on why I ask, the inherent slowness of installing any of dependencies via Flatpak (and Snap) despite its delta update functionality, is ill-suited for my network connection. It's also unusable on my LINX 1010, where the lack of available storage space means that the huge Flatpak runtimes are infeasible to install. Compiling from source would be no different.

Even if that wasn't problematic, I doubt that anyone considers Flatpak to be a *solution*, per se. It's more of a stopgap, when even an accessibility feature, as basic as FontConfig anti-aliasing preferences, don't propogate into the sandbox.

**Reply by leoniscsem** [reply]
My team and me are actually considering to build a pipeline for maintaining a `.deb` PPA for the two current Ubuntu LTS versions (24.04 and 260.4). I do not know much about Fedora or `.rpm` packaging, but might this be a bit of help? If you could be the lab guinea pig for testing on Fedora, we could get our hands on rpm packages as well. (I am very not much of a fan of Snap/Flatpak either)

**Reply by beedell-roke** [reply]
> If you could be the lab guinea pig for testing on Fedora, we could get our hands on RPM packages, as well.

@leoniscsem, I'll be *glad* to test! I believe that many organisations utilise openSUSE's OBS to easily create packages for multiple platforms, if of use.
